core/Objects/CmfApp.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.IO;
+using System.IO.Abstractions;
+using System.Linq;
+using System.Xml.Serialization;
+using Cmf.CLI.Core.Enums;
+using Cmf.CLI.Utilities;
+using Newtonsoft.Json;
+
+namespace Cmf.CLI.Core.Objects.CmfApp;
+
+/// <summary>
+/// CMF App data
+/// </summary>
+public record AppData
+{
+    public string id { get; set; }
+    public string version { get; set; }
+    public string name { get; set; }
+    public string author { get; set; }
+    public string description { get; set; }
+    public string targetFramework { get; set; }
+    public string licensedApplication { get; set; }
+    public string icon { get; set; }
+}
+
+public record Framework
+{
+    [XmlAttribute("version")]
+    public string Version { get; set; }
+}
+
+public record LicensedApplication
+{
+    [XmlAttribute("name")]
+    public string Name { get; set; }
+}
+
+public record Image
+{
+    [XmlAttribute("file")]
+    public string File { get; set; }
+}
+
+[XmlRoot("App")]
+public record CmfAppV1
+{
+    [XmlAttribute("id")] public string Id { get; set; }
+    [XmlAttribute("version")] public string Version { get; set; }
```

Comment:
Even though we discussed using the Version object it seemed to me that it would bring complexity and inconsistency, since other classes/methods are still using string Version, such as Bump in PackageTypeHandler and CmfPackage